src/components/molecules/UiStepper/_internal/UiStepperStep.vue

```diff
@@ -153,6 +152,17 @@ const itemAttrs = computed<ButtonAttrsProps | TextAttrsProps>(() => (isVisitedSt
     }
   }
 
+  &--is-current {
+    --button-color: var(--color-text-body);
+    --button-hover-color: var(--color-text-body);
+    --button-active-color: var(--color-text-body);
+    --button-font: var(--font-body-1-thick);
```

Comment:
```suggestion
    --button-font: var(--font-body-1-thick);
    --button-letter-spacing: var(--letter-spacing-body-1-thick)
```
;)

Reply:
Shouldn't letter spacing be part of font variable? Feels inseparable.